src/index.ts

```diff
@@ -215,8 +222,9 @@ export default class VideoReporter extends WdioReporter {
 
     if (!this.isCucumberFramework && this.options.filenamePrefixSource === 'test') {
       this.testNameStructure.push(suite.title.replace(/ /g, '-').replace(/-{2,}/g, '-'))
+      this.#setRecordingPath()
     }
-    this.#setRecordingPath()
+
```

Comment:
We are we not calling this in Cucumber?

Reply:
no, it's called onSuiteStart instead because that's where scenarios start, so there's one video per scenario.  When this is onTestStart for cucumber, it makes a new folder for every line in the scenario